e2e/tests/test_cases/physical_node_reboot.robot

```diff
@@ -0,0 +1,52 @@
+*** Settings ***
+Documentation    Physical node reboot
+...              https://longhorn.github.io/longhorn-tests/manual/pre-release/node-not-ready/node-restart/physical-node-reboot/
+
+Test Tags    manual_test_case
+
+Resource    ../keywords/common.resource
+Resource    ../keywords/deployment.resource
+Resource    ../keywords/persistentvolumeclaim.resource
+Resource    ../keywords/statefulset.resource
+Resource    ../keywords/volume.resource
+Resource    ../keywords/workload.resource
+Resource    ../keywords/host.resource
+
+Test Setup    Set test environment
+Test Teardown    Cleanup test resources
+
+*** Variables ***
+${LOOP_COUNT}    1
+${RETRY_COUNT}    300
+${RETRY_INTERVAL}    1
+
+*** Test Cases ***
+Physical Node Reboot With Attached Deployment
+    Given Create persistentvolumeclaim 0 using RWO volume
+    And Create persistentvolumeclaim 1 using RWX volume
+    And Create deployment 0 with persistentvolumeclaim 0
+    And Create deployment 1 with persistentvolumeclaim 1
+    And Record workloads attached node    deployment 0    deployment 1
```

Comment:
It's complicated because you want to test everything in a single test case.

You can make `RWX` and `RWO` be a variable too. Like:
```
*** Variables ***
${VOLUME_TYPE}    RWO

*** Test Cases ***
Physical Node Reboot With Attached Deployment
Given Create persistentvolumeclaim 0 using ${VOLUME_TYPE} volume
```

And only dealing with a single workload in a test case, so the test case can be simplified.

No need to explicitly call a keyword to record the volume node:
```
And Record workloads attached node deployment 0 deployment 1
```
Instead, like what you implemented in the previous PR. you can embed it in a keyword:
```
Power off volume node of ${workload_kind} ${workload_id}
${last_volume_node} = get_volume_node ${volume_name}
Set Test Variable ${last_volume_node}
```

Move this logic to a more general keyword like:
```
Create deployment ${deployment_id} with persistentvolumeclaim ${claim_id}
```

Then the volume node recording will be automatic.